.github/workflows/buildAndTest.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         mkdir build
         cd build
         cmake $GITHUB_WORKSPACE/externals/llvm-project/llvm -GNinja \
-          -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_BUILD_TYPE=Asan \
```

Comment:
CMake doesn't know about ASan per se. See https://llvm.org/docs/CMake.html - LLVM_USE_SANITIZER:STRING

Reply:
Ahh, I'm not familiar with ASan so I've just been browsing stack overflow 😅. Thanks for the tip!